webdriver/tests/bidi/input/perform_actions/wheel.py

```diff
@@ -85,7 +86,18 @@ async def test_scroll_iframe(
     await bidi_session.input.perform_actions(
         actions=actions, context=top_context["context"]
     )
-    events = await get_events(bidi_session, top_context["context"])
+
+    events = []
+
+    # Chrome requires some time to process the event from the iframe, so we wait for it.
+    async def wait_for_events(_):
+        nonlocal events
+        events = await get_events(bidi_session, top_context["context"])
+        return len(events) > 0
+
+    wait = AsyncPoll(bidi_session, timeout=0.5)
+    await wait.until(wait_for_events)
```

Comment:
```suggestion
    # Chrome requires some time to process the event from the iframe, so we wait for it.
    async def wait_for_events(_):
        return len(await get_events(bidi_session, top_context["context"])) > 0

    wait = AsyncPoll(bidi_session, timeout=0.5)
    await wait.until(wait_for_events)
    events = await get_events()
```

?

Reply:
This approach gets the events twice. I'd prefer the approach above

Reply:
I'm of the opposite opinion, as relying on side effects of waiting is not great either. We could add `def get_event_count` and only wait for a number of events before getting the events once.